src/models/game.py

```diff
@@ -20,15 +45,56 @@ class Game:
     current_player_id: t.Optional[int]
     step_count: int
     winner_id: t.Optional[int]
-    field: t.Union[str, list]
+    field: t.Union[str, t.List[chr]]
     current_state: str
+    owner_mark: chr = 'X'
+    opponent_mark: chr = '0'
+
+    def _game_loop(self, field: t.List[str]):
+        for condition in WIN_CONDITIONS:
+            if all(field[cell_id] == self._current_mark() for cell_id in condition):
+                self.winner_id = self.current_player_id
+                self.current_state = State.DONE.value
+                break
+
+    def _check_win(self):
+        one_line_field = [colum for line in self.field for colum in line]
+        if one_line_field.count('') == 0:
+            self.current_state = State.DONE.value
+        else:
+            self._game_loop(one_line_field)
+
+    def _invert_player(self):
+        if self.current_player_id == self.owner_id:
+            self.current_player_id = self.opponent_id
+        else:
+            self.current_player_id = self.owner_id
+
+    def _current_mark(self) -> chr:
+        if self.current_player_id == self.owner_id:
+            return self.owner_mark
+        return self.opponent_mark
 
     def set_opponent(self, opponent: User):
         self.opponent_id = opponent.id
         self.current_player_id = random.choice(
             [self.owner_id, self.opponent_id]
         )
+        self.owner_mark, self.opponent_mark = 'X', '0'
```

Comment:
Seems like it can be removed according to next lines
https://github.com/qerdcv/ttt-online/pull/33/files#diff-eed504ec13914227ed78e749c32e6823d3d99ce0083aa64733e99adc9e4bb9b6R51